### PR TITLE
Adding more types, some fixes & improvements

### PR DESCRIPTION
**Pull request by vespakoen**
Hey,

Thanks you so much for this project and the xml-to-typescript tool, they are really nice!

Some things I did:
1. I dumped my PremierePro types and added them
1. I added the AfterEffects types from [atarabi/aftereffects.d.ts](https://github.com/atarabi/aftereffects.d.ts)
@atarabi Is this alright with you?
1. Allow `Bounds` to be a number[]
1. Adding $ and $.global types to the Photoshop
1. Lastly, I was wondering if it's a good idea to use the app identifiers for the folder names?
([A list is available here](https://github.com/Adobe-CEP/CEP-Resources/blob/master/CEP_8.x/Documentation/CEP%208.0%20HTML%20Extension%20Cookbook.md#applications-integrated-with-cep))

### CODE REVIEW

File: global/index.d.ts [hunk 1]
```diff
@@ -5,16 +5,6 @@
...
-/**
- * The Infinity global property is a predefined variable with the value for infinity.
- */
-declare var Infinity: number;
-
-/**
- * The NaN global property is a predefined variable with the value NaN (Not-a-Number), as specified by the IEEE-754 standard.
- */
-declare var NaN: number;
-
```
Comment (pravdomil) [inline global/index.d.ts]: why you have removed Infinity and NaN?

File: JavaScript/index.d.ts [hunk 2]
```diff
@@ -1,1455 +1,3 @@
-/**
```
Comment (pravdomil) [inline JavaScript/index.d.ts]: why this file is removed?

**Comment by pravdomil** [comment]
1) where you have fond the xml documentation file for Premiere?
2) I have contacted atarabi
3) can you please provide a sample code
4) those types are already defined in [this file](https://github.com/pravdomil/types-for-adobe/blob/master/JavaScript/index.d.ts)
5) how would the new file structure look like?


please remove the AfterEffects/2017/.DS_Store file

File: global/index.d.ts (hunk 1 above)
Comment (vespakoen) [reply global/index.d.ts]: I just checked and I found out I was using the `"lib": ["es5"]` option, so it must have collided with that and I have removed those as a result.

Good catch =)

File: JavaScript/index.d.ts (hunk 2 above)
Comment (vespakoen) [reply JavaScript/index.d.ts]: Same reason as the last comment, will fix in new PR's

**Comment by vespakoen** [comment]
Sorry for the mess =)

1). I installed PPRO and used your tool to convert it from xml -> typescript, just dumped the resulting file in there, not tested it yet.
2). Cool!
3). Sample code for what exactly?
4). I think we should move the $ object definition (or $.global if possible) to each individual app's definition so we can change `readonly global: Object;` to point to a more meaningful object? 
5). types-for-adobe/AEFT/2015.3/index.d.ts / types-for-adobe/PPRO/2017/index.d.ts etc

I'll split it up and make new PR's, I guess I'll wait until we have a "go" from atarabi and know how you want to proceed with #4 ?

**Comment by pravdomil** [comment]
thanks for splitting this pull request

2) we have go from atarabi can you please make a pull request for after effects types?
5) I would prefer Premiere over PPRO, Premiere is more human readable

**Pull request closed without merging**